examples/complete/terraform.tfvars.example

```diff
@@ -0,0 +1,6 @@
+org_primary_account = "012345678901"
+external_id         = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+cf_api_key          = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+hackerone_api_token = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
+
+slack_webhook_urls = ["https://hooks.slack.com/services/XXXXXXXXXX/XXXXXXXXXXXXXXXXX"] # dev
```

Comment:
Are these tests planning to test slack, hackerone, cloudflare, etc ? If not, I'd pull these values.

Reply:
I think it's important to keep these in - initially the test will be a Terraform plan / apply - and it's a more comprehensive test if it includes some options such as CloudFlare and HackerOne. Slack notifications are in any case a core functionality of the system.

I'm also planning for now to deploy to my existing test environment, and I don't want to lose the CloudFlare / HackerOne / Slack functionality!